qtconsole/console_widget.py

```diff
@@ -752,12 +752,17 @@ def _get_font(self):
         """
         return self._control.document().defaultFont()
 
+    def _get_font_width(self, font=None):
+        if font is None:
+            font = self.font
+        font_metrics = QtGui.QFontMetrics(font)
+        return font_metrics.horizontalAdvance(' ') if hasattr(font_metrics, 'horizontalAdvance') else font_metrics.width(' ')
```

Comment:
This is too much for a single line. Please use a proper `if` to organize this code better, like this:

```python
if hasattr(font_metrics, 'horizontalAdvance'):
    return font_metrics.horizontalAdvance(' ')
else:
    return font_metrics.width(' ')
```

Reply:
Fixed.